docs/index.md

```diff
@@ -1,8 +1,8 @@
 # Autonolas Developer Documentation
 
-Welcome to the Autonolas Developer Documentation pages.
+Welcome to the Autonolas Developer Documentation pages. These pages aim at providing in-depth, technical background about the [Autonolas Ecosystem](#the-autonolas-ecosystem).
```

Comment:
Tbh I don't think we need this line. Generally it's not considered good practice to welcome in text – the design of the site should provide this "welcoming feeling". And the second sentence is implicit in the concept of docs